### PR TITLE
desktop/windows: remove windows installer

### PR DESCRIPTION
**Pull request by tessr**
This removes the Windows installer and a reference to it in the Github issue template.
It does not remove references to it in other documentation, because people may
still be running the app, which links back to that documentation (e.g. license files).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 Thank you for reporting an issue! Please read the following before submitting:
 
-Github issues are for reporting specific bugs with Chain Core Developer Edition. If you have general questions about Chain Core or questions regarding building an application on Chain Core, please visit our developer support center: 
+Github issues are for reporting specific bugs with Chain Core Developer Edition. If you have general questions about Chain Core or questions regarding building an application on Chain Core, please visit our developer support center:
 https://support.chain.com
 
-Enterprise customers can reach out to their support contact directly.
```
Comment (tessr) [inline .github/ISSUE_TEMPLATE.md]: My editor automatically stripped out these whitespaces. Is that ok? I don't think it makes any difference

**Comment by tessr** [comment]
I don't think we want to modify the change logs? But maybe we should?

**Comment by croaky** [comment]
I think those two lines are currently broken links b/c this PR deletes the referenced `desktop/windows/CHANGELOG.md` files. https://github.com/chain/chain/blob/main/docs/1.2/core/reference/changelog.md

**Comment by tessr** [comment]
Ah! Good catch. I'll delete them, then.

**Comment by tessr** [comment]
Actually - those both point to the `1.2-stable` branch, and I was planning on landing this on `main`. But maybe I should land it on `1.2-stable`?

**Comment by croaky** [comment]
Ah, another good catch. Reviewing how we deploy the docs https://github.com/chain/chain/tree/main/docs#uploading-the-docs it looks like we deploy from each of the `-stable` branches. So we should make these changes on `main`, `1.0-stable`, `1.1-stable`, and `1.2-stable`?

**Pull request closed without merging**